community/knative/charts/build/templates/logging-cm.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+data:
+  loglevel.controller: info
+  loglevel.creds-init: info
+  loglevel.git-init: info
+  loglevel.webhook: info
+  zap-logger-config: |
+    {
+      "level": "info",
+      "development": false,
+      "sampling": {
+        "initial": 100,
+        "thereafter": 100
+      },
+      "outputPaths": ["stdout"],
+      "errorOutputPaths": ["stderr"],
+      "encoding": "json",
+      "encoderConfig": {
+        "timeKey": "",
+        "levelKey": "level",
+        "nameKey": "logger",
+        "callerKey": "caller",
+        "messageKey": "msg",
+        "stacktraceKey": "stacktrace",
+        "lineEnding": "",
+        "levelEncoder": "",
+        "timeEncoder": "",
+        "durationEncoder": "",
+        "callerEncoder": ""
+      }
+    }
+kind: ConfigMap
+metadata:
+  name: config-logging
```

Comment:
The same `metadata.name` is used in charts/serving/templates/config-maps.yaml - resources need to have unique names among their kind.

Reply:
We will fix this in next commit

Reply:
given these are in separate namespaces there is no conflict with names